python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -1207,13 +1209,43 @@ def test_get_description_batch(arctic_library):
         [ReadInfoRequest("symbol1", as_of=0), ReadInfoRequest("symbol2", as_of=0), ReadInfoRequest("symbol3", as_of=0)]
     )
 
-    assert infos[0].date_range == (datetime(2018, 1, 1), datetime(2018, 1, 6))
-    assert infos[1].date_range == (datetime(2019, 1, 1), datetime(2019, 1, 6))
-    assert infos[2].date_range == (datetime(2020, 1, 1), datetime(2020, 1, 6))
+    assert infos[0].date_range == tuple(
```

Comment:
Why have all these assertions changed?

Reply:
Discussed in person. Add a ticket for the issue this fixes, and mark in the PR description that this PR closes it as well.

Reply:
I mentioned the ticket that created the issue, since I realised that no ticket was created to address it at the end